### ISSUE TITLE
Backend Functions

### ISSUE THREAD

**Issue by abhaikollara** [abhaikollara/ktorch#1]
# Backend functions to be completed
- [x] variable
- [x] constant
- [ ] is_keras_tensor
- [x] placeholder
- [x] shape
- [x] int_shape
- [x] ndim
- [x] dtype
- [x] eval
- [x] zeros
- [x] ones
- [x] eye
- [x] zeros_like
- [x] ones_like
- [x] identity
- [x] random_uniform_variable
- [x] random_normal_variable
- [x] count_params
- [x] cast
- [x] update
- [x] update_add
- [x] update_sub
- [x] moving_average_update
- [x] dot
- [ ] batch_dot
- [x] transpose
- [ ] gather
- [x] max
- [x] min
- [x] sum
- [x] prod
- [x] cumsum
- [x] cumprod
- [x] var
- [x] std
- [x] mean
- [x] any
- [x] all
- [x] argmax
- [x] argmin
- [x] square
- [x] abs
- [x] sqrt
- [x] exp
- [x] log
- [x] logsumexp
- [x] round
- [x] sign
- [x] pow
- [x] clip
- [x] equal
- [x] not_equal
- [x] greater
- [x] greater_equal
- [x] less
- [x] less_equal
- [x] maximum
- [x] minimum
- [x] sin
- [x] cos
- [ ] normalize_batch_in_training
- [ ] batch_normalization
- [x] concatenate
- [x] reshape
- [x] permute_dimensions
- [ ] resize_images
- [ ] resize_volumes
- [ ] repeat_elements
- [ ] repeat
- [x] arange
- [ ] tile
- [x] flatten
- [ ] batch_flatten
- [x] expand_dims
- [x] squeeze
- [ ] temporal_padding
- [ ] spatial_2d_padding
- [ ] spatial_3d_padding
- [x] stack
- [x] one_hot
- [ ] reverse
- [x] get_value
- [x] batch_get_value
- [x] set_value
- [x] batch_set_value
- [x] get_variable_shape
- [x] print_tensor
- [ ] function
- [ ] gradients
- [ ] stop_gradient
- [ ] rnn
- [ ] switch
- [ ] in_train_phase
- [ ] in_test_phase
- [x] relu
- [x] elu
- [x] softmax
- [x] softplus
- [x] softsign
- [ ] categorical_crossentropy
- [ ] sparse_categorical_crossentropy
- [ ] binary_crossentropy
- [x] sigmoid
- [x] hard_sigmoid
- [x] tanh
- [x] dropout
- [x] l2_normalize
- [ ] in_top_k
- [ ] conv1d
- [ ] conv2d
- [ ] conv2d_transpose
- [ ] separable_conv2d
- [ ] conv3d
- [ ] pool2d
- [ ] pool3d
- [ ] bias_add
- [x] random_normal
- [x] random_uniform
- [x] random_binomial
- [x] truncated_normal
- [ ] ctc_label_dense_to_sparse
- [ ] ctc_batch_cost
- [ ] ctc_decode
- [ ] map_fn
- [ ] foldl
- [ ] foldr
- [ ] local_conv1d
- [ ] local_conv2d
